### PR TITLE
docs: garden is now landscape

### PR DESCRIPTION
**Pull request by jamesacklin**
Updates the README.md and DEVELOPMENT.md docs to reflect the current state of the world.

@bonbud-macryg, would be thrilled to have a once-over.

### CODE REVIEW

File: DEVELOPMENT.md [hunk 1]
```diff
@@ -52,9 +52,9 @@ Groups and Talk are distributed via the Urbit network by way of a [glob](https:/
 1. Create or launch an urbit using the -F flag.
 2. On that urbit, if you don't already have a desk to run from, run `|merge %work our %base` to create a new desk and mount it with `|mount %work`.
 3. Now the `%work` desk is accessible through the host OS's filesystem as a directory of that urbit's pier ie `~/zod/work`.
-4. From the `ui` directory you can run `rsync -avL --delete dist/ ~/zod/work/groups` where `~/zod` is your fake urbit's pier.
+4. From the `ui` directory you can run `rsync -avL --delete dist/ ~/zod/work/groups` and `rsync -avL --delete desk/ ~/zod/work/` where `~/zod` is your fake urbit's pier.
```
[the remark this answers is not in the document]
Comment (bonbud-macryg) [reply DEVELOPMENT.md:55]: As instructed this gets me the error `clay: no files match /mar/webmanifest/hoon`.

**Review by bonbud-macryg — COMMENTED** [review]
Some errors here before I got to the `-make-glob` step that stopped me before. I was able to get past Deploying Step 4 last time but I don't have a note of what I did to get past it.

**Comment by arthyn** [comment]
@bonbud-macryg made some changes here that should address all your comments, and hopefully is clearer

**Review by bonbud-macryg — COMMENTED** [review]
The purpose of the different sections of the doc is much clearer, thanks @arthyn. I got through the initial setup and the `## Deploying` section with no real issues, and that seems like the important thing. (Only problem there was that it failed to fetch a glob hosted I uploaded to GitHub, but that's unrelated to this.)

***

I'm still having trouble with the rest of the `## Fakezod Development` section, now around Step 5 where there's no `desk-dev`, so I can't follow this exactly and end up `rsync`ing in files from all over the place.

I got `%landscape` compiling eventually. Haven't got `%groups` compiling with the latest set of instructions; I figured it out a few commits ago but can't remember what I did. The furthest I've got getting `%groups` to compile is an error `clay: no files match /lib/docket/hoon`.

The original question I asked in Tlon Local was whether this step-by-step process was worth updating at all or whether everyone should just use Bouncer. Given the new context that it's unlikely people would need to build these desks from scratch at all, is it?

For context this has been a couple hours of trial and error on my end since before I asked @jamesacklin about it, but if it's important to accommodate devs who can't use Bouncer I can do another run through this and take more detailed notes on what I'm having to do differently from the instructions.

**Comment by arthyn** [comment]
@bonbud-macryg following these updated directions with a fresh fake zod booted only with the solid pill aka only %base, worked for me. feel free to DM me on network at ~nocsyx-lassul if you need help.

**Pull request merged**